deepwell/src/endpoints/page.rs

```diff
@@ -72,6 +73,26 @@ pub async fn page_get_direct(
     }
 }
 
+pub async fn page_get_score(
+    ctx: &ServiceContext<'_>,
+    params: Params<'static>,
+) -> Result<Option<GetPageScoreOutput>> {
+    let GetPageReferenceDetails {
+        site_id,
+        page: reference,
+        details: _,
```

Comment:
Is there not a non-details version of the struct we can use here?

Reply:
Doesn't look like it. I'll make one then